pkgs/top-level/all-packages.nix

```diff
@@ -29651,6 +29651,8 @@ with pkgs;
 
   migu = callPackage ../data/fonts/migu { };
 
+  minecraftia = callPackage ../data/fonts/minecraftia { };
```

Comment:
Please move the package to `pkgs/by-name/mi/minecraftia/package.nix` and drop this line, thanks.